src/modules/AggregateRepository.js

```diff
@@ -167,22 +158,33 @@ export default class AggregateRepository {
     if (loadedEvents.length <= 0) {
       return false;
     }
-
     return loadedEvents[loadedEvents.length - 1].id;
   }
 
   applyEntityEvents(loadedEvents, entity) {
-
     loadedEvents.forEach(event => {
-
       const type = event.eventType.split('.').pop();
-
       const applyMethod = this.getApplyMethod(entity, type);
 
       applyMethod.call(entity, event);
     });
-
   }
 
+  find(entityId, options) {
```

Comment:
This should be like other methods and find({entityClass, ....})

Reply:
Resolved with: https://github.com/eventuate-clients/eventuate-client-nodejs/pull/45/commits/726e8d6663662ee029d2e21c1932787feaa962fd